phpBB/styles/prosilver/theme/common.css

```diff
@@ -1251,6 +1251,22 @@ ul.linklist:after,
 	margin-left: 58px;
 }
 
+.badge {
+	border-radius: 10px;
+	opacity: 0.8;
+	text-align: center;
+	white-space: nowrap;
+	font-size: 9px;
```

Comment:
don't use ems stick with pixels. the smaller size is a stylistic one its usually 75% smaller than the corresponding text/icon size.
https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss#L740

I rounded up from 8.25 or 11 * 75%